gnome-initial-setup/pages/live-chooser/eos-setup-live-user

```diff
@@ -178,6 +165,18 @@ class SetupLiveUser(object):
         self.update(GS_SCHEMA, ALLOW_UPDATES, GLib.Variant('b', False))
 
 
+def install_desktop_file():
+    """eos-installer is shipped in all images, but its desktop file is
+    (deliberately) only present in the GDM greeter's environment. For
+    image-booted live sessions, we symlink it into place."""
```

Comment:
nitpick: PEP257 would require a short description (ending with a period) on the first line, followed by a longer description, with the closing triple-parens on the following line.